.bash_profile

```diff
@@ -82,6 +82,25 @@ done
 export GOPATH=/go
 export PATH=$PATH:$GOPATH/bin:./bin/linux_amd64/:./vendor/bin:
 
+# Enable pyenv
+#   Search home directory first and then system, allow pyenv
+#     installed in home to override that of system
+#   KDK has system pyenv installed in /usr/local
+if [[ -d "/usr/local/pyenv/bin" ]]; then
+    export PATH="/usr/local/pyenv/bin:$PATH"
+fi
+if [[ -d "$HOME/.pyenv/bin" ]]; then
+    export PATH="$HOME/.pyenv/bin:$PATH"
+fi
+if  [[ -n ${PS1:-''} ]] && which pyenv &>/dev/null; then
```

Comment:
Nit: There's an extra space in here. No biggie.

Reply:
Pyenv will be installed in /usr/local/pyenv on the KDK.  And this will automatically point to it *if* it exists. If the user already has a local pyenv install that will override the system.

Pulumi install can be intergrated into the KDK, potentially.  I might